src/lightbox/Lightbox.js

```diff
@@ -71,6 +71,10 @@ export default function Lightbox(props: Props) {
       <PhotoView
         source={resource}
         style={[styles.img, { width: windowWidth }]}
+        // Doesn't seem to do anything on iOS:
+        //   https://github.com/alwx/react-native-photo-view/issues/62
+        //   https://github.com/alwx/react-native-photo-view/issues/98
+        // TODO: Figure out how to make it work.
         resizeMode="contain"
```

Comment:
Huh, I see! Indeed, when I try on Android (with the current beta) I get exactly the "pillarboxed" behavior that I'd hope for. It's just on iOS that I don't, and the image stays full size.

Hmm. I've been doing this testing mainly with an image that's a screenshot from a similar phone, so its natural size is the screen size. What if the image's natural size is much larger, or smaller? Does this issue in PhotoView mean that on iOS it'll stay too large, or too small?

Reply:
Here are some screenshots from the iOS simulator, for [an extremely small image](https://chat.zulip.org/#narrow/stream/7-test-here/topic/pictures/near/1111014), and [an extremely large image](https://chat.zulip.org/#narrow/stream/7-test-here/topic/pictures/near/1111013). I've included screenshots of portrait and landscape mode, but it's important to note that, in both cases, I started out in portrait mode, since that seems to make a difference.

<img width="400" alt="Screen Shot 2021-01-29 at 3 33 32 PM" src="https://user-images.githubusercontent.com/22248748/106324672-8ea04400-6247-11eb-89e8-fda7e1fc009a.png">
<img height="400" alt="Screen Shot 2021-01-29 at 3 33 22 PM" src="https://user-images.githubusercontent.com/22248748/106324674-8f38da80-6247-11eb-9766-1cc986d920c7.png">
<img width="400" alt="Screen Shot 2021-01-29 at 3 33 07 PM" src="https://user-images.githubusercontent.com/22248748/106324675-8f38da80-6247-11eb-9589-c247983c6693.png">
<img height="400" alt="Screen Shot 2021-01-29 at 3 32 57 PM" src="https://user-images.githubusercontent.com/22248748/106324676-8fd17100-6247-11eb-9360-73baa915b807.png">

Reply:
I think the native component sets its dimensions when it first initializes, but it doesn't update them at all afterward.

Reply:
> I think the native component showing the image sets its dimensions when it first initializes, but it doesn't update them at all afterward.

Cool, yeah, that seems to be what's happening. So it appropriately scales the image at the start, but doesn't update that when the component gets resized.